pygeoapi/linked_data.py

```diff
@@ -126,10 +137,10 @@ def jsonldify_collection(cls, collection):
             cls.config['server']['url'],
             collection['id']
         ),
-        "name": collection['title'],
-        "description": collection['description'],
+        "name": l10n.translate(collection['title'], locale_),
+        "description": l10n.translate(collection['description'], locale_),
```

Comment:
I think using [`@language`](https://json-ld.org/spec/ED/json-ld-syntax/20120522/#string-internationalization) construct is better practice; I don't think the JSON-LD document should be sensitive to the requested language which seems more suited for _human reading_.

Reply:
Thanks for reviewing and bringing this up. I'm not too familiar with JSON-LD (yet) and I wasn't aware of its language support, but this definitely seems a good approach. I will look into it, thanks! 👍